### PR TITLE
Updated 2 incidents

### PR DESCRIPTION
**Pull request by ContributingThrowaway**
Added tentative date and what looks like an original source to incident CA-LosAngeles-30, and added information/sources and fixed date on CA-Compton-1.

### CODE REVIEW

File: reports/California.md [hunk 1]
```diff
@@ -141,6 +140,22 @@ id: ca-longbeach-1
...
+Footage shows a protestor sitting on the curb. They are bleeding from a wound to the head. Other protestors are attempting to treat them. Greg Doucette says that, per the journalist who submitted the video, police shot first, then gave the dispersal order.
+
+tags: shoot, projectiles
+
+id: ca-losangeles-30
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268052132252602368
+* https://twitter.com/chick_in_kiev/status/1266867554041188353
```
Comment (emarcey) [inline reports/California.md:154]: Am I misunderstanding? This just seems like a megathread on antifa stuff?

File: reports/California.md [hunk 2]
```diff
@@ -141,6 +140,22 @@ id: ca-longbeach-1
...
+* https://twitter.com/andrewcurryla/status/1266905784585052160
```
Comment (emarcey) [inline reports/California.md:155]: How did you determine this is the same incident?

File: reports/California.md (hunk 1 above)
Comment (ContributingThrowaway) [reply reports/California.md:154]: Quite correct. Thanks for noticing. I had two tweets open in different tabs and must have copy-pasted a link to the wrong one. Correcting.

File: reports/California.md (hunk 2 above)
Comment (emarcey) [reply reports/California.md:155]: Hey @ContributingThrowaway, I wanted to circle back to this before approving

Comment (emarcey) [reply reports/California.md:155]: Okay nevermind, I see now from the fixed link

**Pull request merged**